src/lib.rs

```diff
@@ -63,19 +86,20 @@ impl Texture {
             mipmap_filter: FilterMode::Linear,
             lod_min_clamp: -100.0,
             lod_max_clamp: 100.0,
-            compare: CompareFunction::Always,
```

Comment:
why changed?

Reply:
`CompareFunction::Always` would indicate that it's a comparison sampler just with a comparison op that always succeeds. `None` signals that it's not a comparison sampler.